src/bindings/js/node/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ add_library(${PROJECT_NAME} SHARED
 target_include_directories(${PROJECT_NAME} PRIVATE 
     "${node-api-headers_SOURCE_DIR}/include"
     "${node-addon-api_SOURCE_DIR}"
-    "${CMAKE_CURRENT_SOURCE_DIR}/include"
+    "${CMAKE_CURRENT_SOURCE_DIR}/.."
```

Comment:
Why not `${CMAKE_CURRENT_SOURCE_DIR}/.`?

Reply:
Here we specify the directory the compiler uses to search for include files. 
If it's `${CMAKE_CURRENT_SOURCE_DIR}/.`  include paths won't contain the information that the header file is from `node` directory